shared/src/test/diff/gadt/Exp1.mls

```diff
@@ -33,12 +33,18 @@ fun f(e) = if e is
 :e // TODO support
 fun f(e) = if e is
   Pair['a, 'b](l, r) then [l, r]
-//│ ╔══[ERROR] illegal pattern
+//│ ╔══[ERROR] unknown pattern Pair‹'a, 'b›(l, r,)
 //│ ║  l.35: 	  Pair['a, 'b](l, r) then [l, r]
 //│ ╙──      	  ^^^^^^^^^^^^^^^^^^
-//│ fun f: anything -> error
+//│ ╔══[ERROR] identifier not found: l
+//│ ║  l.35: 	  Pair['a, 'b](l, r) then [l, r]
+//│ ╙──      	                           ^
+//│ ╔══[ERROR] identifier not found: r
+//│ ║  l.35: 	  Pair['a, 'b](l, r) then [l, r]
+//│ ╙──      	                              ^
+//│ fun f: anything -> [error, error]
 //│ Code generation encountered an error:
-//│   if expression was not desugared
+//│   unresolved symbol l
```

Comment:
We can actually accept this kind of pattern with a warning message: type parameters `'a` and `'b` will be ignored.